src/helix-ui/styles/modules/button.less

```diff
@@ -47,6 +47,10 @@
       margin: 0;
     }
   }
+
+  hx-busy:last-child {
```

Comment:
Zoom review: Add comment re: why this needs to be last and that a better implementation is coming.